### PR TITLE
BoxDecoration.borderRadius and RoundedRectangleBorder.borderRadius RTL

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @abarth

File: packages/flutter/lib/src/painting/box_decoration.dart [hunk 1]
```diff
@@ -123,7 +123,7 @@ class BoxDecoration extends Decoration {
   ///
   /// Applies only to boxes with rectangular shapes; ignored if [shape] is not
   /// [BoxShape.rectangle].
-  final BorderRadius borderRadius;
+  final BorderRadiusGeometry borderRadius;
```
Comment (gspencergoog) [inline packages/flutter/lib/src/painting/box_decoration.dart]: Sure you don't want to also rename borderRadius to borderRadiusGeometry?

**Comment by gspencergoog** [comment]
LGTM

**Review by gspencergoog — APPROVED** [review]

File: packages/flutter/lib/src/painting/box_decoration.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter/lib/src/painting/box_decoration.dart]: Yes. That would be a breaking change, for one. But also it's inconsistent with how we do this kind of thing elsewhere. The fact that the "FooGeometry" classes exist is mostly a detail most users will never care about.

**Pull request merged**